src/test/java/org/openrewrite/staticanalysis/ReplaceWeekYearWithYearTest.java

```diff
@@ -0,0 +1,82 @@
+package org.openrewrite.staticanalysis;
+
+import org.junit.jupiter.api.Test;
+import org.junitpioneer.jupiter.Issue;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+@SuppressWarnings("SuspiciousDateFormat")
+public class ReplaceWeekYearWithYearTest implements RewriteTest {
```

Comment:
There's no need for public test classes or methods; best to remove those to avoid any confusion about whether the classes or methods are used elsewhere.
```suggestion
class ReplaceWeekYearWithYearTest implements RewriteTest {
```